src/tools/clippy/clippy_utils/src/qualify_min_const_fn.rs

```diff
@@ -176,6 +176,9 @@ fn check_rvalue<'tcx>(
             // FIXME(dyn-star)
             unimplemented!()
         },
+        Rvalue::Cast(CastKind::Transmute, _, _) => {
+            Err((span, "transmute can attempt to turn pointers into intos, so is unstable in const fn".into()))
```

Comment:
`int`s?

Reply:
We allow transmute in const fn nowadays

Reply:
This is a clippy copy of something old, so I'm inclined to leave it erroring, same as it probably did 3 years ago when this code was actually maintained.  Or I could even just `unimplemented!()` it, like `DynStar` is above it.